packages/xrpl/src/models/transactions/common.ts

```diff
@@ -366,3 +369,104 @@ export function parseAmountValue(amount: unknown): number {
   }
   return parseFloat(amount.value)
 }
+
+// 64 * 2
+const MAX_CREDENTIAL_TYPE_LENGTH = 128
+/**
+ * Check a CredentialType for formatting errors
+ *
+ * @param tx A transaction to check the CredentialType for errors
+ * @throws Validation Error if the formatting is incorrect
+ */
+export function validateCredentialType(tx: Record<string, unknown>): void {
```

Comment:
Would this be a more reusable helper function if it kept the format of `is{type}` but took a max length? e.g. `isHex(length)`

Reply:
So keeping it specific to credential lists, but adding the list length as a parameter? Definitely doable, wasn't sure if we had use cases for different lengths in the future (all of them have 8 max for now)